src/main/java/com/todoist/pojo/Utils.java

```diff
@@ -8,4 +13,26 @@ static boolean equals(Object a, Object b) {
     static int clamp(int val, int min, int max) {
         return val < min ? min : (val > max ? max : val);
     }
+
+    public static <T> Set<T> unmodifiableSet(Collection<? extends T> s) {
+        if (s != null) {
+            return Collections.unmodifiableSet(new HashSet<T>(s));
+        } else {
+            return Collections.emptySet();
```

Comment:
We should probably use `Collections. unmodifiableSet(Collections.emptySet())` (we can cache this as a static constant?), so that any changes to the empty set are consistent with any changes to a non-empty set (ie. crash).